tests/algorithms/sample_algorithm_test.cpp

```diff
@@ -0,0 +1,10 @@
+#include "gtest/gtest.h"
+
+TEST(Sample_tests, addition_test) {
+    int addend_a = 2;
```

Comment:
Did you mean "added"? Why not just "a"?

Reply:
Google addend word bro

Reply:
Still, if you want to be precise why didn't you name it as "addend_int_variable_a_equals_two"?